client/templates/kontakt/kontakt.html

```diff
@@ -0,0 +1,47 @@
+<template name="kontakt">
+
+  <div class="menu-sub-view">
+    <div class="houseoffam-logo-mobile">
+      <a href="/" class="menu-nav-hover"> HOUSE OF FAM</a>
+    </div>
+    <div class="menu-nav-sub-wrapper">
+      <div class="menu-nav-sub nettbutikk-sub">
+        <a href="http://fam-irvoll.myshopify.com/collections/frontpage" class="menu-nav-hover"> NETTBUTIKK </a>
+      </div>
+      <div class="menu-nav-sub kontakt-sub">
+        <a href="/kontakt" class="menu-nav-hover">
+          KONTAKT
+        </a>
+      </div>
+      <div class="menu-nav-sub om-sub">
+        <a href="/om" class="menu-nav-hover">
+        OM
+        </a>
+      </div>
+    </div>
+
+    <div class="houseoffam-logo-sub">
+      <a href="/">
+      HOUSE OF FAM
+      </a>
+    </div>
+  </div>
+
+<div class="post">
+
+    <div class="kontaktmanager">
+
+      <div id="manager">
+      <p> All kontakt sendes til </p>
+    </div>
+
+     <br>
+
+    <div id="managername">
+      Ryan Ludlow
+    </div>
+     <a href="mailto:ryan@brandentevents.com"> <div id="manageremail"> ryan@brandentevents.com </div> </a>
+    </div>
+
+  </div> <!-- post -->
+</template>
```

Comment:
All these lines repeating the header are redundant (violating DRY). The way to do it is to tell the router that it should wrap this route using a the `blogLayout` @dagvonkr  is not good to merge. Do you want to correct this yourself or I do it?